src/fragments/lib/in-app-messaging/js/overview.mdx

```diff
@@ -0,0 +1,23 @@
+<Callout >
+
+In-App Messaging is currently supported only for **React Native**.
+
+</Callout>
+
+
+Amplify In-App Messaging helps you better engage your users with messages they can see while actively using your application. Send targeted messages to your defined user segments or even trigger contextual messages based on user behavior. Once you are up and running with In-App Messaging, you’ll be able to create messages that look native to your application and deliver them to your users all without additional code changes.
+
+The different types of messages that can be shown to users are:
+
+- **Top Banner** – a message that appears as a banner at the top of the page.
+- **Bottom Banner** – a message that appears as a banner at the bottom of the page.
+- **Middle Banner** – a message that appears as a banner in the middle of the page.
+- **Full Screen** – a message that covers entire screen.
+- **Modal (Coming soon!)** – a message that appears in a window in front of the page.
+- **Carousel** – a horizontally scrollable layout of up to five unique messages.
+
+## How it works
+
+Unlike traditional push notifications, in-App messages are synced to your user’s device locally so that when an analytics event triggers them, they can be displayed instantly. An added benefit to in-app messages being locally stored is that, once synced, in-app messages can work even in offline use cases.
```

Comment:
Inconsistent casing  of "In-App Messages"
```suggestion
Unlike traditional push notifications, In-App Messages are synced to your user’s device locally so that when an analytics event triggers them, they can be displayed instantly. An added benefit to In-App Messages being locally stored is that, once synced, In-App Messages can work even in offline use cases.
```